src/main/java/com/uber/jenkins/phabricator/coverage/XmlCoverageProvider.java

```diff
@@ -112,6 +112,14 @@ private void parse(Set<String> includeFiles, Set<File> reports) throws SAXExcept
         }
     }
 
+    private static Long getLongValue(NamedNodeMap attrs, String attr) {
+        return Math.round(Double.valueOf(attrs.getNamedItem(attr).getTextContent()));
+    }
+
+    private static Integer getIntValue(NamedNodeMap attrs, String attr) {
+        return Math.round(Float.valueOf(attrs.getNamedItem(attr).getTextContent()));
```

Comment:
Same here

Reply:
This change was to fix numberformat exception. I can make the error clearer

Reply:
:+1: